js/apps/admin-ui/maven-resources/theme/keycloak.v2/admin/messages/messages_en.properties

```
@@ -341,7 +341,7 @@ postBrokerLoginFlowAlias=Post login flow
 refreshTokenMaxReuse=Refresh Token Max Reuse
 partialExportHeaderText=Partial export allows you to export realm configuration, and other associated resources into a json file.
 clientScopes=Client scopes
-loadingRealms=Loading realms…
+loadingRealms=Loading realms\u2026
```

If you need Unicode you can add it directly, we load these files as UTF-8.
```suggestion
loadingRealms=Loading realms…
```

---

Sorry, while IntelliJ loads the file using UTF-8, it performs those changes automatically. I don't know why ...

---

It's likely for historical reasons, property files were not UTF-8 back in the day.